modules/eks/eks.tf

```diff
@@ -50,7 +52,19 @@ module "cluster" {
       subnets                       = var.subnetworks
       additional_security_group_ids = [aws_security_group.nodes.id]
       cpu_credits                   = "unlimited" # Avoid t2/t3 throttling
-      kubelet_extra_args            = trimsuffix(chomp(lookup(node_pool, "kubelet_extra_args")), ",")
+      kubelet_extra_args            = replace(trimsuffix(chomp(lookup(node_pool, "kubelet_extra_args")), ","), "\n", " ")
+      tags = [
+        {
+          "key"                 = "k8s.io/cluster-autoscaler/${var.cluster_name}"
+          "value"               = "owned"
+          "propagate_at_launch" = true
+        },
+        {
+          "key"                 = "k8s.io/cluster-autoscaler/enabled"
+          "value"               = "true"
+          "propagate_at_launch" = true
+        },
+      ]
```

Comment:
These tags are required for cluster-autoscaler, isn't it?

Reply:
Yes, they are.

Reply:
https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html